src/modules/fancyzones/editor/FancyZonesEditor/Models/MainWindowSettingsModel.cs

```diff
@@ -231,7 +232,7 @@ public LayoutModel UpdateSelectedLayoutModel()
             {
                 foreach (LayoutModel model in CustomModels)
                 {
-                    if (model.Uuid == currentApplied.ZonesetUuid.ToUpperInvariant())
+                    if (string.Equals(model.Uuid, currentApplied.ZonesetUuid, StringComparison.OrdinalIgnoreCase))
```

Comment:
The invariant logic was removed here... Shouldn't this be InvariantCultureIgnoreCase?

Reply:
as it's uuid, I believe it won't make a difference. But to be sure let's keep it as it was